src/acceptor.rs

```diff
@@ -23,11 +48,13 @@ impl TlsAcceptor {
     /// This is typically used after a new socket has been accepted from a
     /// `TcpListener`. That socket is then passed to this function to perform
     /// the server half of accepting a client connection.
-    pub async fn accept<S>(&self, stream: S) -> Result<TlsStream<S>, Error>
+    pub async fn accept<S>(&self, stream: S) -> Result<TlsStream<S>, native_tls::Error>
     where
         S: AsyncRead + AsyncWrite + Unpin,
     {
-        handshake(move |s| self.0.accept(s), stream).await
+        handshake(move |s| self.0.accept(s), stream)
```

Comment:
This also works:
```suggestion
        let stream = handshake(move |s| self.0.accept(s), stream).await?;
        Ok(stream)
```
`?` calls `Into` automatically :sparkles: